pages.pt_BR/linux/top.md

```diff
@@ -0,0 +1,23 @@
+# top
+
+> Mostra informações em tempo real sobre processos em execução.
+
+- Iniciar top:
+
+`top`
+
+- Esconde processos inativos:
+
+`top -i`
+
+- Mostra processos apenas de um usuário específico:
+
+`top -u {{user_name}}`
+
+- Mostra processos de um ou mais PID específico(s), separado por vírgula:
+
+`top -p {{PID1,PID2,PID3}}`
+
+- Mostra Ajuda sobre os comandos disponíveis:
+
+`?`
```

Comment:
This looks weird. Is this a command param?

Reply:
Opa! Estamos no contexto pt/br haha, me confundi. Esse ponto de interrogação no final me confundiu um pouco. Isso é um parâmetro do comando? Eu vi que na [página em inglês](https://github.com/tldr-pages/tldr/blob/master/pages/linux/top.md) também existe esse ponto de interrogação mas não entendi qual o uso.

Reply:
@schneiderl, nao é um parâmetro, mas como em a tldr-page de `cmus` parece que tem sido adicionados as teclas que servem como atalhos.